integrations/torchvision/README.md

```diff
@@ -0,0 +1,202 @@
+
+### SparseML CLI
+
+SparseML's CLI enables you to kick-off sparse training workflows with various utilities like creating training pipelines, dataset loading, checkpoint saving, metric reporting, and logging handled for you.
+
+To get started, we just need to a couple key arguments:
+```bash
+sparseml.image_classification.train \
+    --checkpoint-path [CHECKPOINT-PATH] \
+    --recipe [RECIPE-PATH] \
+    --dataset-path [DATASET-PATH]
+```
+
+- `--checkpoint-path` specifies the starting model to use in the training process. It can either be a local path to a PyTorch checkpoint or a SparseZoo stub (which SparseML uses to download a PyTorch checkpoint).
+
+- `--dataset-path` specifies the dataset used for training. It must be a local path to a dataset in the ImageNet format (see CLI tutorials for more details).
+
+- `--recipe` specifies the sparsity related parameters of the training process. It can either be a local path to a YAML recipe file or a SparseZoo stub (which SparseML uses to download a YAML recipe file). The `recipe` is the key to enabling the sparsity-related algorithms implemented by SparseML (see the CLI tutorials for more details on recipes).
+
+For full usage, run:
+```bash
+sparseml.image_classification --help
+```
+
+### SparseML Python API
+
+For additional flexibility, SparseML has a Python API that also enables you to add sparsification to a native PyTorch training loop.
+
+Just like the CLI, the Python API uses YAML-based recipes to encode the parameters of the sparsification process, allowing youto add SparseML with just a few lines of code.
+
+The `ScheduleModifierManager` class is responsible for parsing the YAML recipes and overriding the standard PyTorch model and optimizer objects, 
+encoding the logic of the sparsity algorithms from the recipe. Once you call `manager.modify`, you can then use the model and optimizer as usual, as SparseML abstracts away the complexity of the sparsification algorithms.
+
+The workflow looks like this:
+
+```python
+# typical model, optimizer, dataset definition
+model = Model()
+optimizer = Optimizer()
+train_data = TrainData()
+
+# parse recipe + edit model/optimizer with sparsity-logic
+from sparseml.pytorch.optim import ScheduledModifierManager
+manager = ScheduledModifierManager.from_yaml(PATH_TO_RECIPE)
+optimizer = manager.modify(model, optimizer, len(train_data))
+
+# PyTorch training loop, using the model/optimizer as usual
+
+# clean-up
+manager.finalize(model)
+```
+
+See the tutorials for full working examples of the Python API.
+
+## Quick Start: Sparse Transfer Learning
+
+### Overview
+
+Sparse Transfer is quite similiar to the typical transfer learing process used to train NLP models, where we fine-tune a pretrained checkpoint onto a smaller downstream dataset. With Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified checkpoint and maintain sparsity while the training process occurs.
```

Comment:
Sparse Transfer is similar to the typical transfer learning process used to train NLP models, where we fine-tune a pre-trained checkpoint onto a smaller downstream dataset.